scripts/mk_all.sh

```diff
@@ -1,6 +1,33 @@
 #! /usr/bin/env bash
+
```

Comment:
I personally would start with a one-sentence description, something like: "create or update a file with all imports in a given directory" or list of directories. (Or shamelessly copying from #11874, something like "Generate a file importing all the files of a Lean folder. By default, it generates the files for the libraries `Mathlib`, `Mathlib/Tactic`, `Archive` and `Counterexamples`.")